### DISCUSSION TITLE
Documentation for Zotonic's core javascript libraries?

### DISCUSSION THREAD

**Discussion by geoidesic** [zotonic/zotonic#3918] [Q&A]
There is documentation for Cotonic but not for the other libraries that are included in the default site:

```django
{% lib
	"js/modules/jstz.min.js"
	"js/apps/zotonic-wired.js"
	"js/apps/z.widgetmanager.js"
	"js/modules/livevalidation-1.3.js"
	"js/modules/z.dialog.js"
	"js/modules/jquery.loadmask.js"
%}
```

Can you please consider documenting these? Or at least describe them here in the discussions.

**Comment by mworrell** [comment]
Yes, we should.  Will have a go at it.
